codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/restxml/AWSRestXMLHttpResponseBindingErrorGenerator.kt

```diff
@@ -34,11 +37,9 @@ class AWSRestXMLHttpResponseBindingErrorGenerator : HttpResponseBindingErrorGene
 
                 openBlock("extension ${ctx.symbolProvider.toSymbol(ctx.service).name}Types {", "}") {
```

Comment:
Converts the error factories for services & operations to use readers & closures instead of Swift Codable.